mycli/main.py

```diff
@@ -960,7 +960,8 @@ def get_last_query(self):
                     'by default.'))
 # as of 2016-02-15 revocation list is not supported by underling PyMySQL
 # library (--ssl-crl and --ssl-crlpath options in vanilla mysql client)
-@click.option('-v', '--version', is_flag=True, help='Output mycli\'s version.')
+@click.option('-V', '--version', is_flag=True, help='Output mycli\'s version.')
+@click.option('-v', '--verbose', is_flag=True, help='Verbose output.')
```

Comment:
I wonder if it's necessary to switch -V by -v: -V for verbose and -v for version?
the code is great for me